frontend/src/app/components/search/search.component.ts

```diff
@@ -106,6 +107,7 @@ export class SearchComponent implements OnInit {
       this.years = year.filter(this.onlyUnique).sort();
 
       this.loading = false;
+      setTimeout(_=>{this.searchBox.focus();});
```

Comment:
```js
new Promise(_ => this.searchBox.focus());
```

or even
```js
new Promise(this.searchBox.focus);
````
Just a promise, not a promise with anything was what I was thinking of